packages/address/README.md

```diff
@@ -74,6 +74,8 @@ await getTranslationKey('US', 'province'); // => "state"
 await getTranslationKey('JA', 'province'); // => "prefecture"
 await getTranslationKey('CA', 'zip'); // => "postalCode"
 await getTranslationKey('US', 'zip'); // => "zipCode"
+await getTranslationKey('CA', 'address2'); // => "aptUnitNumber"
+await getTranslationKey('JA', 'address2'); // => "aptSuiteEtc"
```

Comment:
Can you actually add all of the possible values that we already list in [country-service](https://github.com/Shopify/country-service#translation-keys)?

Reply:
Currently, we only have two values in country-service

Reply:
I know but we don't say explicitly all of the possible values neither for `zip` or `province`.
I would basically copy paste the block of doc we have there.

Reply:
Oh I see what you mean now!